scripts/config/templates/nginx.conf.erb

```diff
@@ -138,76 +111,15 @@ http {
 
   rewrite ^/embed/(.*)$ /$1.html last;
 
-  # the speedy_search_assignment cookie forces nginx to always send that user to that arm of the test
-  # the speedy_should_identify cookie tells the clients to identify the assignment in Rudderstack
-  # 1. Speedy Search - SSR
-  if ($speedy_search_condition = "on_<%= ssr_frontend_host %>_/search") {
-    rewrite ^(.*)$ /speedy-search/$1;
+  location ^~ /search {
+    proxy_pass <%= ssr_frontend_host %>;
   }
-  
+
   # redirect to speedy home page in ssr-frontend
   location = / {
     proxy_pass <%= ssr_frontend_host %>/$is_args$args;
   }
 
-  location ^~ /speedy-search {
-    rewrite ^/speedy-search/(.*)$ $1 break;
-    add_header Set-Cookie "speedy_search_assignment=<%= ssr_frontend_host %>; Max-Age=31536000; Path=/";
-    add_header Set-Cookie "speedy_should_identify=$speedy_should_identify_default_true; Max-Age=31536000; Path=/";
```

Comment:
We'll need to remove this check in SSR so don't stop identify events.

Reply:
Isn't this just for the `identify`s for the A/B test user property `AB_useSpeedySearch`

Reply:
You're right